src/stream/src/executor/join/builder.rs

```diff
@@ -156,6 +157,29 @@ impl<const T: JoinTypePrimitive, const SIDE: SideTypePrimitive> JoinChunkBuilder
         }
     }
 
+    pub fn post_process(c: StreamChunk) -> StreamChunk {
+        let mut c = StreamChunkMut::from(c);
```

Comment:
Please add some comment about this.

Also is it possible to add this in `append_row_matched` etc.

Reply:
only do it on the `JoinChunkBuilder` but not for the `JoinStreamChunkBuilder`

Reply:
Why not adding comments :(

Reply:
I forgot. 😶‍🌫️